lib/puppet/type/openldap_database.rb

```diff
@@ -83,6 +83,26 @@ def insync?(is)
           "{SSHA}" + Base64.encode64("#{Digest::SHA1.digest("#{should}#{salt}")}#{salt}").chomp == is
         when /^\{SHA\}.+/
           "{SHA}" + Digest::SHA1.hexdigest(should) == is
+        when /^\{SSHA(256|384|512)\}.+/
```

Comment:
No, It is not. 
cause the regex you mentioned checks the '}' character at the end of the SSHA string.
I checked various cases, but It was OK.

If you are do not agree with me, please give me a way.
Cause I am a just beginner of Ruby thing.